frontend/src/gamestate/experimentalGame.js

```diff
@@ -17,26 +17,49 @@ const tileMap = new Map();
 function MainGame() {
     const [tileMap, setTileMap] = useState(new Map());
     const [tilesInitialized, setTilesInitialized] = useState(false);
-
-    const backgroundMusic = useRef(null);
-    useEffect(() => {
-        const audio = backgroundMusic.current;
-        audio.volume = 0.2;
-        const handleCanPlay = () => {
-          audio.play().catch((error) => {
-            console.log("Autoplay was prevented, retrying with mute...", error);
-            audio.muted = true; // Mute the audio if autoplay is blocked
-            audio.play();
-          });
+    const [playerScore, setPlayerScore] = useState(0);
+    const [botScore, setBotScore] = useState(0);
+
+    const BackgroundMusic = ({ src }) => {
```

Comment:
Consider moving the audio into a UseEffect, with an empty array as a dependency, so there is always one Audio component created.

Reply:
Try this:
```javascript
const audioRef = useRef(null);
const [isPlaying, setIsPlaying] = useState(false);
const [volume, setVolume] = useState(0.5);

useEffect(() => {
  audioRef.current = new Audio(src);
  audioRef.current.volume = volume;

  return () => {
    audioRef.current.pause();
    audioRef.current = null; 
  };
}, [src]);
```

Reply:
Already tried this.